client/cmd/dexc-desktop/pkg/common.sh

```diff
@@ -0,0 +1,19 @@
+
+# This file defines common variables to be source'd by the various build scripts
+# in this directory.
+
+# pick up the release tag from git
+VER=$(git describe --tags --abbrev=0 --always | sed -e 's/^v//')
```

Comment:
Hmm. If I do

```Bash
git fetch upstream --tags
git checkout -b v0.6.3 v0.6.3
git describe --tags --abbrev=0 --always | sed -e 's/^v//'
```
I get
```Bash
0.6.2
```

Reply:
Does `upstream` point to `github.com/decred/dcrdex`?  I'm getting the correct tag on a fresh clone.

```bash
$ git clone https://github.com/decred/dcrdex
Cloning into 'dcrdex'...
...
$ cd dcrdex
$ git fetch origin --tags
$ git checkout -b v0.6.3 v0.6.3
Switched to a new branch 'v0.6.3'
$ git describe --tags --abbrev=0 --always | sed -e 's/^v//'
0.6.3
```